src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObject = {};
+  const lines = sourceString
+    .split('')
+    .filter((val) => val !== '\n')
+    .join('')
+    .split(';');
+
+  lines.forEach((word) => {
+    const cleanWord = word.trimStart().trimEnd();
+    const dividedLine = cleanWord.split(':');
+
+    if (typeof dividedLine[1] !== 'undefined') {
+      const key = dividedLine[0].trimStart().trimEnd();
+      let value = dividedLine[1].trimStart().trimEnd();
+
+      if (value.includes(',')) {
+        value = value.split(',').join(',\n');
+      }
+
+      convertedObject[key] = value;
```

Comment:
```suggestion
    const [ key.trim(), value.trim() ]= cleanWord.split(':');

    if (value) { // means if it presents or it's NOT undefined
      if (value.includes(',')) {
        value = value.split(',').join(',\n');
      }

      convertedObject[key] = value;
```
We can simplify it using array destructuring